config/dev.exs

```diff
@@ -11,7 +11,7 @@ config :scorify_elixir, ScorifyElixirWeb.Endpoint,
   debug_errors: true,
   code_reloader: true,
   check_origin: false,
-  watchers: [yarn: ["run", "watch", cd: Path.expand("../assets", __DIR__)]]
+  watchers: [npm: ["run", "watch", cd: Path.expand("../assets", __DIR__)]]
```

Comment:
Does this command need to use `npm` specifically and not `yarn`?
I don't have an opinion on which one is better; but perhaps if we check in `yarn.lock` and consistently use `yarn`, let's just use `yarn` commands?